core/graph/src/main/java/uk/gov/gchq/gaffer/graph/Graph.java

```diff
@@ -283,10 +283,22 @@ public String getGraphId() {
         return store.getGraphId();
     }
 
+    /**
+     * @return the StoreProperties for this Graph.
+     */
+    public StoreProperties getStoreProperties() {
+        return store.getProperties();
+    }
+
     public GraphLibrary getLibrary() {
         return library;
     }
 
+    public void setLibrary(final GraphLibrary library) {
```

Comment:
I don't think this should be a public method. Everything should be set in the Graph Builder.